spell/psm-checklists.md

```diff
@@ -63,89 +68,87 @@
               - [ ] EITHER code diff check for each individual file
               - [ ] OR bytecode verification tool such as `forge verify-bytecode`
         - [ ] Deployed contract matches the code in the [repo](https://github.com/makerdao/dss-lite-psm/)
-        - [ ] Deployer is no longer the `owner`
-        - [ ] `MCD_PAUSE_PROXY` is the `owner` of the contract (i.e. `require(MomLike(MOM).owner() == MCD_PAUSE_PROXY`))
-    - `LitePsmJob`
+        - Constructor params:
+            - [ ] ilk is named `LITE-PSM-{TOKEN_SYMBOL}-A` (i.e. `LITE-PSM-USDC-A`)
+            - [ ] `gem` matches the expected token address
+            - [ ] `daiJoin` matches `MCD_JOIN_DAI` from the chainlog
+            - [ ] `pocket` matches the pocket address in the Exec Sheet
+        - [ ] Deployer no longer has privileged access (`wards(deployer) == 0`)
+        - [ ] `MCD_PAUSE_PROXY` has been authed (i.e. `require(WardsLike({LITE_PSM}).wards(MCD_PAUSE_PROXY) == 1)`)
+        - [ ] Sanity check: `DssLitePsm` has `type(uint256).max` approval to spend `gem` on behalf of `pocket`
+    - [ ] `LitePsmJob`
         - [ ] Uses the default Solidity version in [`dss-cron`](https://github.com/makerdao/dss-cron)
         - [ ] Optimizer is **enabled**
         - [ ] Deployed contract matches the code in the [repo](https://github.com/makerdao/dss-cron)
         - Constructor params:
             - [ ] `_sequencer` matches `CRON_SEQUENCER` address from the chainlog
-            - [ ] `_litePsm` matches `DssLitePsm` address
+            - [ ] `_litePsm` matches `{LITE_PSM}` address
             - [ ] `_rushThreshold` matches the Exec Sheet (it might be named as "fill threshold")
             - [ ] `_gushThreshold` matches the Exec Sheet (it might be named as "trim threshold")
             - [ ] `_cutThreshold` matches the Exec Sheet (it might be named as "chug threshold")
 - Initialization
-    - `DssLitePsm`:
+    - [ ] `DssLitePsm`:
         - [ ] `buf` is set to the value specified in the Exec Sheet
         - [ ] `vow` is set to `MCD_VOW` from the chainlog
-        - [ ] `MCD_PAUSE_PROXY` is whitelisted to execute swaps with no fees (i.e. `KissLike(LITE_PSM).kiss(MCD_PAUSE_PROXY)`)
-        - [ ] `DssLitePsmMom` is authed (i.e. `RelyLike(LITE_PSM).rely(MOM)`)
-    - `DssLitePsmMom`:
-        - [ ] The chief (`MCD_ADM`) is set as the `authority` (i.e. `MomLike(MOM).setAuthority(MCD_ADM)`)
-    - `LitePsmJob`:
+        - [ ] `MCD_PAUSE_PROXY` is whitelisted to execute swaps with no fees (i.e. `KissLike({LITE_PSM}).kiss(MCD_PAUSE_PROXY)`)
+        - [ ] `DssLitePsmMom` is authed (i.e. `RelyLike({LITE_PSM}).rely(MOM)`)
+    - [ ] `LitePsmJob`:
         - [ ] Job is added to `CRON_SEQUENCER`
     - [ ] `MCD_VAT`: new ilk is initialized
     - [ ] `MCD_JUG`: new ilk is initialized
-    - `MCD_SPOT`:
+    - [ ] `MCD_SPOT`:
```

Comment:
For 102 and 108 it kind of makes sense to have the check because they are conditional branches, so it you mark it, it means you are following that branch.

Also for visual consistency, maybe we should not be that strict that items with 1 single sub-item should be inlined.